gaffer-as-a-service/gaas-ui/package-lock.json

```diff
@@ -0,0 +1,17483 @@
+{
```

Comment:
Assuming this file is required (it seems **very large**) - copyright header missing.

Reply:
removed